### PR TITLE
Add CHANGELOG support

### PR DESCRIPTION
**Pull request by misteroneill**
This is a pretty barebones implementation, but it automates some of the tedium out of maintaining a CHANGELOG. It also makes the CHANGELOG optional (on by default).

### CODE REVIEW

File: docs/standards.md [hunk 1]
```diff
@@ -120,24 +120,26 @@ __All standard video.js plugins must implement the core set of npm scripts.__
...
-`watch:test` |          | Triggers a build when the test entry point changes.
-`version`    | ✓        | Special handling if Bower support is enabled.
+npm Script    | Optional | Description
+------------- | -------- | -----------
+`build`       |          | Runs all build sub-tasks.
+`build:css`   | ✓        | Builds the Sass entry point.
+`build:js`    |          | Builds the Browserify entry point.
+`build:lang`  | ✓        | Builds language files.
+`build:test`  |          | Builds the test Browserify entry point.
+`change`      | ✓        | Adds a free-form text entry to the CHANGELOG.
```
Comment (gkatsev) [inline docs/standards.md]: would you run it as:
```sh
$ npm run change -- this is the item to add to the changelog
```
?

File: src/app/templates/scripts/_version.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    return false;
+  }
+};
+
+var commands = [];
+
+// If the project has a CHANGELOG, update it for the new release.
+if (hasChangelog()) {
+  commands = commands.concat([
+    'chg release "' + pkg.version + '"',
```
Comment (gkatsev) [inline src/app/templates/scripts/_version.js]: this would get the new version we want?

File: docs/standards.md (hunk 1 above)
Comment (misteroneill) [inline docs/standards.md]: You can do that, but `npm run change` will also produce a prompt via `chg`.

File: src/app/templates/scripts/_version.js (hunk 2 above)
Comment (misteroneill) [inline src/app/templates/scripts/_version.js]: Yeah. This is run as the npm `"version"` script, at which point the `package.json` has been version-bumped, but it has not been committed yet.

File: docs/standards.md (hunk 1 above)
Comment (gkatsev) [inline docs/standards.md]: Ah, cool. It should be documented that that is what happens. It isn't obvious to me.

File: src/app/templates/scripts/_version.js (hunk 2 above)
Comment (gkatsev) [inline src/app/templates/scripts/_version.js]: :+1:

File: docs/standards.md (hunk 1 above)
Comment (misteroneill) [inline docs/standards.md]: Makes sense. :+1:

**Comment by gkatsev** [comment]
LGTM

**Pull request merged**